docs/network_graph_attributes.md

```diff
@@ -4,7 +4,7 @@ The [network graph overview](network_config.md) provides a general summary of Sh
 
 ### Example Graph
 
-Below is an example of a simple network graph in the Shadow-supported GML format (note that GML calls _vertices_ as _nodes_).
+Below is an example of a simple network graph in the Shadow-supported GML format.
```

Comment:
I think Rob had preferred `vertex` over `node` so that people don't get "node" and "host" mixed up, but the GML docs below should probably use "node" since that is the keyword used in GML. Maybe a comment can be added here to make sure the user doesn't confuse a node with a host.

Reply:
Hmm, yeah on second thought I think I'll revert this change and go back to vertex. The mismatch with "node" vs "vertex" in GML is a little unfortunate, but I agree "node" is maybe too ambiguous a term in general.